content/_includes/home/implementations.html

```diff
@@ -14,20 +14,20 @@ <h3 id="In_Production" class="section">In Production: <a href="#In_Production" c
 
 {: .table .table-striped .rows-3}
-|Software|Description|Platform support|
-|--- |--- |--- |
-|[Galaxy](https://galaxyproject.org/)|Web-based platform for data intensive biomedical research.|&#8211;|
-|[cwl-tes](https://github.com/ohsu-comp-bio/cwl-tes)|CWL engine backended by the [GA4GH Task Execution API](https://github.com/ga4gh/task-execution-schemas)|Alicloud, AWS, Google, HPC, Local, Spark, TES|
-|[Xenon](https://nlesc.github.io/Xenon/)|Run CWL workflows using Xenon|[any Xenon backend](https://nlesc.github.io/Xenon/): local, ssh, SLURM, Torque, Grid Engine |
-|[Consonance](https://github.com/Consonance/consonance)|orchestration tool for running SeqWare workflows and CWL tools|AWS, OpenStack, Azure|
-|[AWE](https://github.com/MG-RAST/AWE)|Workflow and resource management system for bioinformatics data analysis.|&#8211;|
-|[yacle](https://github.com/otiai10/yacle)|Yet Another CWL Engine|local|
-|[Calrissian](https://github.com/Duke-GCB/calrissian)|CWL Engine built for Kubernetes|[Kubernetes](https://kubernetes.io/)|
-|[Cromwell](https://github.com/broadinstitute/cromwell)|Cromwell workflow engine|Google, HTCondor, Local, LSF, PBS/Torque, SGE, Slurm, TES|
-|[CWLEXEC](https://github.com/IBMSpectrumComputing/cwlexec)|Apache 2.0 licensed CWL executor for IBM Spectrum LSF, supported by IBM for customers with valid contracts.|[IBM Spectrum LSF](https://developer.ibm.com/storage/products/ibm-spectrum-lsf/#) 10.1.0.3+|
-|[Mariner](https://github.com/uc-cdis/mariner)|"The Gen3 Workflow Execution Service", Apache 2.0 licensed, written in Go, also implements the [GA4GH WES API](https://ga4gh.github.io/workflow-execution-service-schemas)|[Kubernetes](https://kubernetes.io)|
-|[ep3](https://github.com/tom-tan/ep3)|Extremely Pluggable Pipeline Processor| local |
-|[Pegasus](https://pegasus.isi.edu/documentation/reference-guide/cwl-support.html)|Pegasus Workflow Management System|Partial support for importing CWL workflows is [under development](https://pegasus.isi.edu/documentation/manpages/pegasus-cwl-converter.html)|  
-|[StreamFlow](https://streamflow.di.unito.it/)|Workflow Management System for hybrid HPC-Cloud infrastructures|Full support for CWL 1.2 is currently under development|
+|Software|Description|Compliance|Platform support|
+|--- |--- |--- |--- |
+|[Galaxy](https://galaxyproject.org/)|Web-based platform for data intensive biomedical research.||&#8211;|
+|[cwl-tes](https://github.com/ohsu-comp-bio/cwl-tes)|CWL engine backended by the [GA4GH Task Execution API](https://github.com/ga4gh/task-execution-schemas)||Alicloud, AWS, Google, HPC, Local, Spark, TES|
+|[xenonflow](https://github.com/xenon-middleware/xenonflow)|Run CWL workflows using Xenon through a REST api.|![Required](https://badgen.net/https/raw.githubusercontent.com/xenon-middleware/xenon-flow/gh-pages/badges/required.json)|[any Xenon backend](https://xenon-middleware.github.io/xenon/versions/3.1.0/javadoc/): local, ssh, SLURM, Torque, Grid Engine |
```

Comment:
```suggestion
|[xenonflow](https://github.com/xenon-middleware/xenonflow)|Run CWL workflows using Xenon through a REST api.|[![Required](https://badgen.net/https/raw.githubusercontent.com/xenon-middleware/xenon-flow/gh-pages/badges/required.json)](https://github.com/xenon-middleware/xenonflow#cwl-compliance)|[any Xenon backend](https://xenon-middleware.github.io/xenon/versions/3.1.0/javadoc/): local, ssh, SLURM, Torque, Grid Engine |
```
Link to the full results